tests/net/lib/mqtt_sn_client/src/mqtt_sn_client.c

```diff
@@ -92,7 +92,7 @@ static int tp_init(struct mqtt_sn_transport *tp)
 }
```

Comment:
About the commit subjects of the test commits, please change to `tests: net: mqtt...`

Reply:
done